# ManipulatorStateType state
int32 state
JointStateM[] joints
